### DISCUSSION TITLE
能不能再添加一个http/https访问

### DISCUSSION THREAD

**Discussion by Uranusual** [1Remote/1Remote#522] [Ideas]
不想再单独开个浏览器进行http访问后台，希望可以添加进去

**Comment by VShawn** [comment]
我不准备单独添加 http 协议，你可以使用 app 协议把浏览器包括进去，不过用 chrome 似乎效果不太好。

http://1remote.org/usage/protocol/especial/app/

![image](https://github.com/1Remote/1Remote/assets/10143738/eb93b963-6be9-4384-abc9-bf3b97b6d2be)
